instrumentation/github.com/labstack/echo/otelecho/echo_test.go

```diff
@@ -117,3 +119,37 @@ func TestSkipper(t *testing.T) {
 	router.ServeHTTP(w, r)
 	assert.Equal(t, http.StatusOK, w.Result().StatusCode, "should call the 'ping' handler") //nolint:bodyclose // False positive for httptest.ResponseRecorder: https://github.com/timakin/bodyclose/issues/59.
 }
+
+func TestHTTPRouteAttribute(t *testing.T) {
```

Comment:
Until we bring everything back into the main package, this test should be in the `test` subfolder.